Cinema Journal/Model/Networking.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 struct Networking {
     
-    static func fetchData<T>(_ request : APITarget, _ modelType : T.Type) where T: Codable {
+    static func fetchData<T>(_ request : APITarget, _ modelType : T.Type, completionHandler : @escaping ([String : Any]?, Error?) -> Void) where T: Codable {
```

Comment:
Why downcasting to dictionary. You're using generics, you have `T.Type`, when you decode, it'll be `T`. So completion may be `@escaping (T?, Error?) -> Void`